src/main/java/com/openshift/restclient/authorization/TokenAuthorizationStrategy.java

```diff
@@ -38,5 +38,19 @@ public void accept(IAuthorizationStrategyVisitor visitor) {
 		visitor.visit(this);
 	}
 
-
+	@Override
+	public int hashCode() {
```

Comment:
@jcantrill dont want to rely on the token by choice, since I consider 2 authentication strategies with different tokens to be identical